app/main.py

```diff
@@ -1,25 +1,60 @@
+from __future__ import annotations
+from typing import Callable
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Callable, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: SlideLimitationValidator,
+                owner: Callable) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: SlideLimitationValidator, value: int) -> None:
+        if self.min_amount <= value <= self.max_amount:
+            setattr(instance, self.protected_name, value)
```

Comment:
What happens in other way?

Reply:
nothing

Reply:
in other way you must `raise ValueError`
Please fix this